allennlp/training/metric_tracker.py

```diff
@@ -10,27 +10,26 @@ class MetricTracker:
     `state_dict` / `load_state_dict` interface, so that it can be checkpointed along with
     your model and optimizer.
 
-    Some metrics improve by increasing; others by decreasing. Here you can either explicitly
-    supply `should_decrease`, or you can provide a `metric_name` in which case "should decrease"
-    is inferred from the first character, which must be "+" or "-".
+    Some metrics improve by increasing; others by decreasing. You can provide a
+    `metric_name` that starts with "+" to indicate an increasing metric, or "-"
+    to indicate a decreasing metric.
 
     # Parameters
 
+    metric_name : `Union[str, List[str]]`
+        Specifies the metric or metrics to track. Metric names have to start with
+        "+" for increasing metrics or "-" for decreasing ones. If you specify more
+        than one, it tracks the sum of the increasing metrics metrics minus the sum
+        of the decreasing metrics.
```

Comment:
For multitask, it does not make sense to track only one metric for early stopping, so I gave this thing the ability to track the sum of multiple metrics. If you want something fancier you need to write your own derivation of `MultitaskModel` that produces a metric you can track. I'm hoping that this will capture the 90% case though.